tests/test_integration.py

```diff
@@ -17,15 +17,20 @@ def test_missing_manifests():
     Ensure all suites (except a blacklist) are generating manifest information.
     """
     BLACKLIST = (
-        "web-platform-tests",
+        "crashtest",
+        "cppunittest",
+        "geckoview-junit",
+        "jittest",
+        "jsreftest",
+        "reftest",
         "talos",
+        "test-verify-wpt",
+        "web-platform-tests",
+        "web-platform-tests-crashtest",
+        "web-platform-tests-crashtests",
+        "web-platform-tests-reftest",
         "web-platform-tests-reftests",
         "web-platform-tests-wdspec",
-        "web-platform-tests-crashtests",
```

Comment:
The test passes for me by just adding `web-platform-tests-crashtest` and `web-platform-tests-reftest`

Reply:
Is it passing this way? Or are there both `web-platform-tests-reftests` and `web-platform-tests-reftest`?

Reply:
It seems that only the singular ones are the ones passing.

Reply:
This makes sense, Edwin and I recently changed the labels to remove the plural (to match what the WPT harness actually calls them). Sorry I didn't think of this sooner.. for some reason I thought this was happening to normal reftest/crashtest.

Reply:
Yeah I was worried there could still be some of the old ones in the results from the query, but I guess the change has happened enough time ago.